x/dualstaking/types/expected_keepers.go

```diff
@@ -32,6 +32,7 @@ type EpochstorageKeeper interface {
 	ModifyStakeEntryCurrent(ctx sdk.Context, chainID string, stakeEntry epochstoragetypes.StakeEntry, removeIndex uint64)
 	UnstakeHoldBlocks(ctx sdk.Context, block uint64) (res uint64)
 	UnstakeHoldBlocksStatic(ctx sdk.Context, block uint64) (res uint64)
+	GetEpochStartForBlock(ctx sdk.Context, block uint64) (epochStart, blockInEpoch uint64, err error)
```

Comment:
(not needed: see commend where it is used)

Reply:
done. see https://github.com/lavanet/lava/commit/d1e2d3c1afe278bf342ca8f975d03ca5326020c6